core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1288,7 +1285,7 @@ private static boolean validStoragePolicy(final String value) {
    * @return true if this is a valid key,value pair for setting HDFS policy
    */
   public static boolean isValidHdfsPolicy(String key, String value) {
-    if (key == null || !key.startsWith(TABLE_HDFS_POLICY_PREFIX.getKey()))
+    if (key == null)
       return false;
 
     if (key.equals(Property.TABLE_CODING_POLICY.getKey())) {
```

Comment:
Expanding on the property type comment I made, since there is no validation for this prop its type could be a generic string.  The storage policy prop could have a specific property type that validates values using the code in `validStoragePolicy()`

Reply:
Thanks for pointing that out!  I hadn't seen the PropertyType validation code.